pkg/controllers/dynakube/logmonitoring/daemonset/reconciler.go

```diff
@@ -96,7 +97,7 @@ func (r *Reconciler) generateDaemonSet() (*appsv1.DaemonSet, error) {
 	ds, err := daemonset.Build(r.dk, r.dk.LogMonitoring().GetDaemonSetName(), getContainer(*r.dk, tenantUUID),
 		daemonset.SetInitContainer(getInitContainer(*r.dk, tenantUUID)),
 		daemonset.SetAllLabels(labels.BuildLabels(), labels.BuildMatchLabels(), labels.BuildLabels(), r.dk.LogMonitoring().Template().Labels),
-		daemonset.SetAllAnnotations(nil, r.dk.LogMonitoring().Template().Annotations),
+		daemonset.SetAllAnnotations(nil, maputils.MergeMap(r.dk.LogMonitoring().Template().Annotations, r.buildAnnotations(r.dk))),
```

Comment:
do we still need it?

Reply:
Ye, we have to add an annotation to the daemonset so the pods will restart.

Reply:
check my other comment with renaming, then this one would be much easier to udnerstand.